bom/openhab-addons/pom.xml

```diff
@@ -911,7 +911,7 @@
     </dependency>
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
-      <artifactId>org.openhab.binding.modbus.sunspec</artifactId>
+      <artifactId>org.openhab.binding.surepetcare</artifactId>
```

Comment:
You should not replace other binding entries

Reply:
this has been fixed now.